src/signer/src/main/java/ee/ria/xroad/signer/tokenmanager/module/ModuleConf.java

```diff
@@ -51,41 +56,97 @@
     // Maps Type (UID) to ModuleType
     private static final Map<String, ModuleType> MODULES = new HashMap<>();
 
-    private static final String DEFAULT_TOKEN_ID_FORMAT =
-            "{moduleType}{slotIndex}{serialNumber}{label}";
+    private static final String DEFAULT_TOKEN_ID_FORMAT = "{moduleType}{slotIndex}{serialNumber}{label}";
+
+    // <mechanism name, machanism code>
+    public static final Map<String, Long> SUPPORTED_SIGN_MECHANISMS = createSupportedSignMechanismsMap();
```

Comment:
Mutable fields should not be "public static" (Sonar critical level issue). 
Preferably there should be a method for accessing the required data (if any). Exposing the Map makes it possible to modify it outside of the class. All other similar fields here are private.